spyder_notebook/notebookplugin.py

```diff
@@ -12,6 +12,8 @@
 import sys
 
 # Qt imports
+from qtpy import PYQT4
+from qtpy.QtWebEngineWidgets import WEBENGINE

```

Comment:
This import needs to be before the `qtpy.QtWidgets` one, so that they are organized alphabetically.